### PR TITLE
[JENKINS-63798] Fix extensions footer location

### PR DESCRIPTION
**Pull request by yJunS**
See [JENKINS-63798](https://issues.jenkins-ci.org/browse/JENKINS-63798).

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically

### CODE REVIEW

**Comment by daniel-beck** [comment]
@yJunS Could you provide more context and explanation for this change? For example, is it a regression, and if so, from what change? Where does this show as a bug? How will it impact plugins that add a footer?

**Comment by yJunS** [comment]
@daniel-beck Yes, It's a regression from [JENKINS-61496](https://github.com/jenkinsci/jenkins/pull/4599). Before the changes, they are together, but they are separate now, like this:
image

**Comment by daniel-beck** [comment]
@yJunS This looks like a bug that qualifies for LTS backporting. Please file a Jira issue and reference it in this PR, as described in the PR template.

**Comment by yJunS** [comment]
@daniel-beck Okay. I also filed the [JENKINS-63798](https://issues.jenkins-ci.org/browse/JENKINS-63798)

**Review by varyvol — APPROVED** [review]

**Review by alecharp — APPROVED** [review]
Looks pretty simple and effective. Thanks.

**Review by oleg-nenashev — APPROVED** [review]
👍 
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by fqueiruga** [comment]
I would put them inside of the `#footer` div, because that's where many plugins hook up their stuff.

**Review by fqueiruga — CHANGES_REQUESTED** [review]
I'd suggest moving this above (see previous comments)

**Comment by yJunS** [comment]
> I'd suggest moving this above (see previous comments)

@fqueiruga  It will show this location if put them inside of the *#footer* div, It looks weird.
image

**Comment by fqueiruga** [comment]
That's the place for extensions though. I think the translations assitance plugin used that place IIRC

**Comment by yJunS** [comment]
image
@fqueiruga Do you mean to put it on the same level as the *.page-footer__flex-row* div?

**Comment by oleg-nenashev** [comment]
@fqueiruga Hi. Regardless of the discussion, are you blocking the current edition of the pull request? AFAICT, you rather suggest a follow-up improvement. Or is it required for a proper issue fix?

**Comment by fqueiruga** [comment]
I'd say this issue is a won't fix IMO

**Comment by fqueiruga** [comment]
I think we need screenshots of this actually working with plugins hooking up to it, both desktop and mobile.

**Comment by yJunS** [comment]
Before the version 2.235.1:
image
Since the version 2.235.1:
image
After update:
image

**Comment by fqueiruga** [comment]
I would put it before the version and rest api link

**Comment by yJunS** [comment]
> I would put it before the version and rest api link

Okay. I already changed it

**Review by fqueiruga — APPROVED** [review]

**Comment by MRamonLeon** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Review by LinuxSuRen — APPROVED** [review]
LGTM

Thanks a lot for this improvement.

**Comment by oleg-nenashev** [comment]
Restarted CI

**Comment by oleg-nenashev** [comment]
Thanks for your contribution @yJunS !

**Comment by yJunS** [comment]
@oleg-nenashev It's okay. I want to join `jenkinsci` organization. How can I do it?

**Comment by zbynek** [comment]
Looks like this triggered a display bug (related to `translate` extension?)

image

File: core/src/main/resources/lib/layout/layout.jelly [hunk 1]
```diff
@@ -231,6 +231,12 @@ THE SOFTWARE.
         <div class="page-footer__flex-row">
           <div class="page-footer__footer-id-placeholder" id="footer"></div>
 
+          <j:if test="${extensionsAvailable}">
+            <j:forEach var="pd" items="${h.pageDecorators}">
+              <st:include it="${pd}" page="footer.jelly" optional="true" />
```
Comment (timja) [inline core/src/main/resources/lib/layout/layout.jelly:236]: Caused https://github.com/jenkinsci/jenkins/pull/5001

**Pull request merged**